ScratchDjango/Profile/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from ScratchDjango.User.models import User
+from ScratchDjango.User.serializers import UserSerializer
+
+from .models import Profile
+from .validators import validate_image
+
+
+class ProfileSerializer(serializers.Serializer):
+    user = UserSerializer()
+    avatar = serializers.ImageField(validators=[validate_image])
+    bio = serializers.CharField()
+    
+    def create(self, validated_data):
```

Comment:
Can we do this using signal? whenenver a user created his profile is created automatically?